docs/section22/Rule22-2.md

```diff
@@ -0,0 +1,42 @@
+
+# CHW&CW - Rule 22-2  
+
+**Rule ID:** 22-2  
+**Rule Description:** When baseline building requires CHW loop, design CHW Loop return temperature is 56F.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.8 Chilled-water design supply temperature (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13: `PLACEHOLDER`
+
+## Rule Logic:  
+
+- For each fluid loop in B_RMR: `for fluid_loop_b in B_RMR.ASHRAE229.fluid_loops:`
+
+  - Check if loop is cooling type: `if fluid_loop_b.type == "COOLING":`
+
+    - Get child loops under cooling loop: `child_loops_b = fluid_loop_b.child_loops`
+
```

Comment:
I don't think we need child loops as this only applies to primary loop.

Reply:
Removed child loop. Same question as in Rule 22-1, does ASHRAE229. fluid_loops include secondary loop?

Reply:
Confirmed that ASHRAE229. fluid_loops include all fluid loops.